### DISCUSSION TITLE
Migratoin hanging

### DISCUSSION THREAD

**Discussion by megadaltonit** [miguelgrinberg/Flask-Migrate#417] [Q&A]
flask db migrate - this works fine, and gets all the right changes

I then run "flask db upgrade", it finds the file however no SQL changes are done and the connection just hangs.

I've tried killing all the processes in MySQL and re-run but still no luck

**Comment by miguelgrinberg** [comment]
Impossible for me to help. Also note that Flask-Migrate is a thin wrapper, there is no migration logic in this project, migrations are done by Alembic.
